config/routes.rb

```diff
@@ -1,14 +1,22 @@
 Rails.application.routes.draw do
   with_options format: false do |r|
-    r.get '/healthcheck', :to => proc { [200, {}, ['OK']] }
+    r.put "/draft-content/*base_path", to: "content_items#put_draft_content_item"
+    r.put "/content/*base_path", to: "content_items#put_live_content_item"
 
-    r.constraints base_path: %r[/.*] do
-      put "/draft-content/*base_path", to: "content_items#put_draft_content_item"
-      put "/content/*base_path", to: "content_items#put_live_content_item"
-
-      put "/publish-intent/*base_path", to: "publish_intents#create_or_update"
-      get "/publish-intent/*base_path", to: "publish_intents#show"
-      delete "/publish-intent/*base_path", to: "publish_intents#destroy"
-    end
+    r.put "/publish-intent/*base_path", to: "publish_intents#create_or_update"
+    r.get "/publish-intent/*base_path", to: "publish_intents#show"
+    r.delete "/publish-intent/*base_path", to: "publish_intents#destroy"
   end
+
+  get '/healthcheck', :to => proc { [200, {}, ['OK']] }
```

Comment:
Yes, regardless of the format or locale that might be tagged on to the end of `/healthcheck` it's always going to return "OK".